backend/createStaffAndRooms.py

```diff
@@ -49,41 +49,45 @@ def real_date(a_date: str, year=None):
     """Convert string date into python date
 
     Args:
-        a_date (str): date string, 
-            expected format: "Mon - 11/7"
+        a_date (str): date string,
+            expected format: "Mon - 11/7", "11/7"
         year (Optional[int]): year, in 4-digit format
             Allows specification of year, otherwise is in current year at runtime
     Returns:
         date: python `date` object
     """
     year = year or datetime.now().year
-    _, date = a_date.split('-')
+    date_bits = a_date.split("-")
+    date = None
+    if len(date_bits) == 1:
+        date = date_bits[0]
+    elif len(date_bits) == 2:
+        date = date_bits[1]
+    else:
+        raise Exception(f"Unexpected date format {a_date}")
+
     month, day = date.lstrip().split('/')
     return parse_date("%s-%s-%s" % (year, month, day))
 
-def create_rooms_main(rooms_file, is_hardrock=False, force_roombaht=False):
+def create_rooms_main(args):
+    rooms_file = args['rooms_file']
+    hotel = None
+    if args['hotel_name'] == 'ballys':
+        hotel = "Ballys"
+    elif args['hotel_name'] == 'hardrock':
+        hotel = 'Hard Rock'
+    else:
+        raise Exception(f"Unknown hotel name {args['hotel_name']}specified")
+
     rooms={}
     _rooms_fields, rooms_rows = ingest_csv(rooms_file)
-    # validate imported rooms list
-    # rooms_import_list = [RoomPlacementListIngest(**r) for r in rooms_rows]
-    validation_errors = []
     rooms_import_list = []
     for r in rooms_rows:
         try:
             room_data = RoomPlacementListIngest(**r)
             rooms_import_list.append(room_data)
         except ValidationError as e:
-            logger.info(f"Validation error for row {r}: {e}")
-            validation_errors.append(e)
-    if validation_errors:
-        with open("validation_errors.txt", "w") as f:
-            for line in validation_errors:
-                f.write(f"{line}\n")
-    
-    if(is_hardrock):
-        hotel = "Hard Rock"
-    else:
-        hotel = "Ballys"
+            logger.warning("Validation error for row %s", e)
```

Comment:
This diff after merge seems off. I think you want to keep the `is_hardrock` check?